workflow-gateway/templates/service.yaml

```diff
@@ -11,7 +11,6 @@ spec:
   type: {{ .Values.service.type }}
   ports:
     - port: {{ .Values.service.port }}
-      targetPort: http
```

Comment:
you still want this line, basically this file should be unchanged

Reply:
crap, thanks